prism/src/main/java/org/apache/falcon/service/EntitySLAAlertService.java

```diff
@@ -145,8 +145,9 @@ void processSLACandidates(){
                     if (entityType.equalsIgnoreCase(EntityType.PROCESS.name())){
                         store.putSLAAlertInstance(entityName, clusterName, entityType,
                                 nominalTime, true, false);
```

Comment:
Shouldn't the final arg of putSLA... be true, as the critical watermark is breeched too?

Reply:
Agreed.. Made The change..